360-Videos-Metadata/360VideosMetadata.py

```diff
@@ -363,6 +363,15 @@ def tag_copy(in_fh, out_fh, size):
       out_fh: file handle, destination for saved file.
       size: int, amount of data to copy.
     """
+
+    # On 32-bit systems reading / writing is limited to 2GB chunks.
+    # to prevent overflow we reading 64 MB chunks.
```

Comment:
"to prevent overflow we reading 64 MB chunks." -> englishify this better, please :)

"To prevent overflow, read in/write out in 64 MB chunks."